### PR TITLE
Feature UI additions clay

### PR DESCRIPTION
**Pull request by claywatkins**
This PR adds the following:

- Adds colors to the ColorHelper class that we decided upon in our color palette
- Decorates the tableview with some colors 
- Adds and animation to the tableview when loading the cells
- Cleaned up the profile button on the mapview
- mapview on detailview will zoom to the location of where the city is
- Cleaned up and removed comments

### CODE REVIEW

File: labs-ios-starter/Helpers/ColorsHelper.swift [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
 class ColorsHelper {
-    // MARK: TODO -
-    // Add Colors here
-    
-    
+    static let harvestGold = UIColor(red: 223, green: 151, blue: 53)
+    static let brownSugar = UIColor(red: 167, green: 111, blue: 70)
+    static let sonicSilver = UIColor(red: 117, green: 123, blue: 127)
+    static let lightFrenchBeige = UIColor(red: 210, green: 181, blue: 115)
+    static let nonPhotoBlue = UIColor(red: 164, green: 225, blue: 243)
```
Comment (iOSjess) [inline labs-ios-starter/Helpers/ColorsHelper.swift:16]: This is cool. I'd like to see what you did here and how you were able to incorporate it.

File: labs-ios-starter/Storyboards/Map/MapViewController.swift [hunk 2]
```diff
@@ -53,6 +54,14 @@ class MapViewController: UIViewController {
...
     }
     
+    private func layoutProfileButton(){
+        profileButton.layer.cornerRadius = 25
+        profileButton.layer.shadowOffset = CGSize(width: -5, height: 5)
+        profileButton.layer.shadowOpacity = 0.6
+        profileButton.layer.shadowRadius = 5
+        profileButton.layer.shadowColor = UIColor.black.cgColor
+    }
+
```
Comment (iOSjess) [inline labs-ios-starter/Storyboards/Map/MapViewController.swift:64]: This looks much better. Nice touch!

File: labs-ios-starter/Storyboards/Map/MapViewController.swift [hunk 3]
```diff
@@ -97,11 +106,7 @@ extension MapViewController: MKMapViewDelegate {
     
     func mapView(_ mapView: MKMapView, annotationView view: MKAnnotationView, calloutAccessoryControlTapped control: UIControl) {
         let ac = UIAlertController(title: locationName!, message: "What would you like to do?", preferredStyle: .alert)
-        ac.addAction(UIAlertAction(title: "Add To Favorites", style: .default, handler: { (_) in
-            //Handle adding to favorites
-        }))
```
Comment (iOSjess) [inline labs-ios-starter/Storyboards/Map/MapViewController.swift:102]: Good choice removing this unused button.

File: labs-ios-starter/Storyboards/Profile/LocationDetailViewController.swift [hunk 4]
```diff
@@ -58,6 +59,36 @@ class LocationDetailViewController: UIViewController {
...
+                self.detailMapView.addAnnotation(annotation)
+                
+                let coordinate = CLLocationCoordinate2DMake(latitude!, longitude!)
+                let span = MKCoordinateSpan(latitudeDelta: 0.1, longitudeDelta: 0.1)
+                let region = MKCoordinateRegion(center: coordinate, span: span)
+                self.detailMapView.setRegion(region, animated: true)
+            }
+        }
+
+    }
```
Comment (iOSjess) [inline labs-ios-starter/Storyboards/Profile/LocationDetailViewController.swift:91]: I feel like if we had more time we should've moved a lot of this duplicate code into some helper files.

**Review by iOSjess — APPROVED** [review]
Nice work! These are some great additions. The colors are nice and really make the UI more fun.

File: labs-ios-starter/Storyboards/Profile/LocationDetailViewController.swift (hunk 4 above)
Comment (claywatkins) [reply labs-ios-starter/Storyboards/Profile/LocationDetailViewController.swift:91]: I agree

**Pull request merged**